cmd/kubeadm/app/cmd/phases/controlplane.go

```diff
@@ -18,185 +18,155 @@ package phases
-	configutil "k8s.io/kubernetes/cmd/kubeadm/app/util/config"
 	"k8s.io/kubernetes/pkg/util/normalizer"
 )
 
 var (
 	allControlplaneLongDesc = normalizer.LongDesc(`
 		Generates all static Pod manifest files necessary to establish the control plane.
-		` + cmdutil.AlphaDisclaimer)
+		`)
 
 	allControlplaneExample = normalizer.Examples(`
 		# Generates all static Pod manifest files for control plane components, 
 		# functionally equivalent to what generated by kubeadm init.
-		kubeadm alpha phase controlplane all
+		kubeadm init phase control-plane
 
 		# Generates all static Pod manifest files using options read from a configuration file.
-		kubeadm alpha phase controlplane --config masterconfiguration.yaml
+		kubeadm init phase control-plane --config config.yaml
 		`)
 
 	apiServerControlplaneLongDesc = fmt.Sprintf(normalizer.LongDesc(`
 		Generates the static Pod manifest file for the API server and saves it into %s file.
-		`+cmdutil.AlphaDisclaimer), kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.KubeAPIServer, kubeadmconstants.GetStaticPodDirectory()))
+		`), kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.KubeAPIServer, kubeadmconstants.GetStaticPodDirectory()))
 
 	controllerManagerControlplaneLongDesc = fmt.Sprintf(normalizer.LongDesc(`
 		Generates the static Pod manifest file for the controller-manager and saves it into %s file.
-		`+cmdutil.AlphaDisclaimer), kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.KubeControllerManager, kubeadmconstants.GetStaticPodDirectory()))
+		`), kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.KubeControllerManager, kubeadmconstants.GetStaticPodDirectory()))
 
 	schedulerControlplaneLongDesc = fmt.Sprintf(normalizer.LongDesc(`
 		Generates the static Pod manifest file for the scheduler and saves it into %s file.
-		`+cmdutil.AlphaDisclaimer), kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.KubeScheduler, kubeadmconstants.GetStaticPodDirectory()))
+		`), kubeadmconstants.GetStaticPodFilepath(kubeadmconstants.KubeScheduler, kubeadmconstants.GetStaticPodDirectory()))
 )
 
-// NewCmdControlplane returns main command for Controlplane phase
-func NewCmdControlplane() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "controlplane",
+type controlPlaneData interface {
+	Cfg() *kubeadmapi.InitConfiguration
+	ManifestDir() string
+}
+
+// NewControlPlanePhase creates a kubeadm workflow phase that implements bootstrapping the control plane.
+func NewControlPlanePhase(cfg *kubeadmapi.InitConfiguration) workflow.Phase {
+	phase := workflow.Phase{
+		Name:  "control-plane",
 		Short: "Generates all static Pod manifest files necessary to establish the control plane",
 		Long:  cmdutil.MacroCommandLongDescription,
+		Run:   runControlPlane,
 	}
-
 	manifestPath := kubeadmconstants.GetStaticPodDirectory()
-	cmd.AddCommand(getControlPlaneSubCommands(manifestPath, "")...)
-	return cmd
+	subPhases := getControlPlaneSubPhases(cfg, manifestPath)
```

Comment:
If you check at the PR I Iinked, I'm proposing a slightly different approach for doing this.
WDYT?